src/process/impl/ProcessAPIImpl.java

```diff
@@ -60,14 +60,16 @@ public File getOutputFile(DataStorage ds, String filePath, int jobNumber) throws
 			if (f.getPath().equals(filePath + jobNumber)) {
 				outputFile = f;
 				break;
-			} else
+			} else {
 				continue;
+			}
 		}
 
-		if (outputFile.getPath().equals("default"))
+		if (outputFile.getPath().equals("default")) {
 			throw new Exception("Failed to find requested file: filepath " + filePath + " job#: " + jobNumber);
-		else
+		} else {
 			return outputFile;
+		}
 	}
```

Comment:
I think one output is a great idea. This is should also be okay if the user input multiple inputs as well !